### DISCUSSION TITLE
mmkv is not supported by ios

### DISCUSSION THREAD

**Discussion by adhamsameh** [Tencent/MMKV#954] [Q&A]
hi first of all thank you for sharing knowledge for free and wish you all the best
I am working on react-native project which contain a chat room, so i used the **MMKV** to make it reliable and fast real time chat but eventually after I have finished the app run on android perfectly while on ios it shows this errors
![WhatsApp Image 2022-09-21 at 10 22 56 AM](https://user-images.githubusercontent.com/87545909/191571022-0c648dae-00bb-4ff9-a99f-7081ebdfbf10.jpeg)

 I tried to remove the package and run the program it works so it is clear that the problem is from the package so please if any body could help me

**Comment by lingol** [comment]
Duplicate with #955.
